kudos-webapps/src/main/webapp/groovy/social-extension/activity/commons/UIActivityCommentActions.gtmpl

```diff
@@ -22,14 +28,14 @@
           class="v-btn v-btn--icon small mt-0 mb-0 mr-0 ml-0"
           onclick="document.dispatchEvent(new CustomEvent('exo-kudos-open-send-modal', {'detail' : {'id' : '${commentId}', 'type': 'COMMENT', 'parentId': '${activity.id}'}}));event.preventDefault();event.stopPropagation();">
     <div class="v-btn__content">
-      <span class="kudosLabel lightGrey">Kudos</span>
+      <span class="$kudosCommentClass">Kudos</span>
```

Comment:
Shouldn' this button be in primary color even in case there are no kudos already sent ? cc @samehaidi

Reply:
Do you confirm that @mzorai that the kudos button on comment block (level 1 and levels 2) should has primaryColor (blue) regardless the number of kudos sent on the comment ?